test/playlist-selectors.test.js

```diff
@@ -203,11 +203,28 @@ test('simpleSelector limits using resolution information when it exists', functi
     master,
     bandwidth: 4194304,
     playerWidth: 444,
-    playerHeight: 790,
+    playerHeight: 500,
```

Comment:
why change the player height here?

Reply:
For the object-fit case, I wanted to test that it selects the smallest rendition that has both at least player width and player height. With the old player height value there would have been no rendition with greater height than the player height. So I needed a smaller player height to make sure it does not select a too big rendition.

I wanted the both test cases to only differ regarding "object-fit", to make it clear that that's what's causing the different result. Lowering the value here does not change anything since the rendition in chosen based on player width in this case.

Reply:
perhaps it would be better to change behavior in a new test and keep this test as (except for the change to an options object for simpleSelector)